Tests/Fixtures/Generated/Strings/flat-swift4/plurals-advanced.swift

```diff
@@ -0,0 +1,41 @@
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+import Foundation
+
+// swiftlint:disable superfluous_disable_command file_length implicit_return
+
+// MARK: - Strings
+
+// swiftlint:disable function_parameter_count identifier_name line_length type_body_length
+internal enum L10n {
+  /// Plural case 'other': Your %3$@ list contains %1$d items. You should buy them %2$@.
+  internal static func multiplePlaceholdersAndVariablesIntStringString(_ p1: Int, _ p2: Any, _ p3: Any) -> String {
+    return L10n.tr("LocPluralAdvanced", "multiple.placeholders-and-variables.int-string-string", p1, String(describing: p2), String(describing: p3))
+  }
+  /// Plural case 'other': %@ has %d ratings
+  internal static func multiplePlaceholdersAndVariablesStringInt(_ p1: Any, _ p2: Int) -> String {
```

Comment:
Just wanted to be sure: does that work at runtime? I mean, the fact that the first parameter is a `%@` and thus not a number, is Foundation able to pluralise everything properly at runtime with that example?

Looking at the definition of that entry I think it is, because both the `%@` and `%d` are actually variables in the `.stringsdict` and each variable has its pluralisation rule, but wanted to be sure.

That made me realise that maybe the comment for each key should not be `Plural case 'other': {entry.translation}` but should instead just directly be the `NSStringLocalizedFormatKey` of the entry (`"%#@element@ %#@has_rating@"` in this case)? 🤷

Reply:
> Just wanted to be sure: does that work at runtime? I mean, the fact that the first parameter is a `%@` and thus not a number, is Foundation able to pluralise everything properly at runtime with that example?
> 
> Looking at the definition of that entry I think it is, because both the `%@` and `%d` are actually variables in the `.stringsdict` and each variable has its pluralisation rule, but then what value does it use to decide if it should pick zero, one, many or other if you pass a `String`? (in that particular case you only have `other`, but still, the logic seems strange to begin with)

As mentioned in the other reply above: This does not seem to work during runtime and returns `(null)` for that variable. 
> That made me realise that maybe the comment for each key should not be `Plural case 'other': {entry.translation}` but should instead just directly be the `NSStringLocalizedFormatKey` of the entry (`"%#@element@ %#@has_rating@"` in this case)? 🤷

I thought it would be more helpful to have an actual translation in the doc comment than the `NSStringLocalizedFormatKey`, as with the normal `.strings` files. But since we will always have the problems of possibly different translations based on the different plural rules, I can also remove the whole bunch of code that unfurls the format key into this _other_ case translation.